### ISSUE TITLE
Provider disconnect causes extreme logging on subscriber

### ISSUE THREAD

**Issue by carbonin** [2ndQuadrant/pglogical#28]
When the provider node is no longer reachable the subscriber node logs an extremely large amount to the postgresql.log file.

### Versions
PostgreSQL 9.5.2
pglogical 1.0.1
pglogical_output 1.0.1

### Steps to reproduce
- Set up a provider and subscriber pair.
- Kill the PostgreSQL server running on the provider

### Observed behavior

The following group of lines are printed at ~300 times per second to the postgresql.log file on the subscriber server.
```
2016-07-26 19:25:34 GMT::5797b92e.70a5:@:[28837]:LOG:  starting apply for subscription region_0_subscription
2016-07-26 19:25:34 GMT::5797b92e.70a5:@:[28837]:ERROR:  could not connect to the postgresql server in replication mode: could not connect to server: Connection refused
                Is the server running on host "192.168.211.161" and accepting
                TCP/IP connections on port 5432?

2016-07-26 19:25:34 GMT::5797b92e.70a5:@:[28837]:DETAIL:  dsn was: dbname='database' host='hostname' user='username' port='5432'
2016-07-26 19:25:34 GMT::5797b103.184b:@:[6219]:LOG:  worker process: pglogical apply 20633:1658409966 (PID 28837) exited with exit code 1
```

I'm not sure if this is really a bug, but if you don't catch the connection loss quickly this can easily cause tons of disk space to be consumed by postgresql logs very quickly.

Is there a way to configure a timeout between attempts to start the apply process if we can't connect to the provider?

**Comment by PJMODOS** [comment]
I think this got fixed in 1.1.x

**Comment by ringerc** [comment]
1.2.x is out, please reopen if you still see this.
